### PR TITLE
Down command

### PR DESCRIPTION
**Pull request by jacMelloni**

### CODE REVIEW

File: denv.go [hunk 1]
```diff
@@ -74,6 +68,27 @@ func main() {
...
+			os.Exit(1)
+		}
+
+		args := []string{}
+
+		for _, file := range service.Files {
+			args = append(args, "-f")
+			args = append(args, file)
+		}
+		args = append(args, "down")
```
Comment (tobi812) [inline denv.go]: Can we somehow make that part more abstract? It almost looks like the "up" section. What do you think?

Comment (tobi812) [reply denv.go]: For example by extracting it to a separate function.

Comment (jacMelloni) [reply denv.go]: Sounds good, let me see what I can abstract!

Comment (tobi812) [reply denv.go]: Perfecto dude :)

**Review by tobi812 — APPROVED** [review]
Nice one, great refactoring :+1:

**Pull request merged**